src/inc/msquic_winkernel.h

```diff
@@ -24,6 +24,7 @@
 #include <minwindef.h>
 #include <ntstatus.h>
 #include <basetsd.h>
+#include <netioddk.h>
```

Comment:
This is the "public" header that external clients consume. Is there any risk of this breaking their builds?

Reply:
It might if their code conflicts with the definitions or something in the header but netioddk.h is a public header file published in DDK. I don't know what we can do to reduce the risk. Are there really external KM consumers?

Reply:
We don't have external KM customers. I'm just worried about an ingest PR possibly breaking Windows build.